packages/cli/src/commands/test.js

```diff
@@ -1,20 +1,46 @@
 import execa from 'execa'
-import Listr from 'listr'
-import VerboseRenderer from 'listr-verbose-renderer'
 import terminalLink from 'terminal-link'
 
 import { getPaths } from 'src/lib'
 import c from 'src/lib/colors'
 
+const jest = require('jest')
+
+// TODO: Get from redwood.toml
+const sides = ['web', 'api']
```

Comment:
Could we add a `getSides` method to `@redwoodjs/internal` that returns these values?

Reply:
We started that process here: https://github.com/redwoodjs/redwood/pull/633 

Last we talked the next steps were for you to revisit the tests for `internal`. Also fine if we end up doing something else and scratch this PR -- e.g. would `project-structure` be the right place for something like this now?

Reply:
@thedavidprice Yup! `structure` will most likely be the source of truth for this.

Reply:
I'm not sure where to get these from now, any pointers? @peterp @thedavidprice

Reply:
@RobertBroersma Maybe you could add it over here: https://github.com/redwoodjs/redwood/blob/d7d50b941156225a9cc33cca82bc34b521d7300d/packages/structure/src/model/RWProject.ts#L157 and just return these two static values?

Reply:
I think I did the thing, please verify